thoth/adviser/report.py

```diff
@@ -76,10 +79,18 @@ def add_product(self, product: Product) -> bool:
 
     def to_dict(self) -> Dict[str, Any]:
         """Convert pipeline report to a dict representation."""
+        stack_info: List[Dict[str, Any]] = []
+        stack_info_metadata = os.getenv("THOTH_ADVISER_METADATA")
+        if stack_info_metadata:
+            try:
+                stack_info = ((json.loads(stack_info_metadata).get("thoth.adviser") or {}).get("stack_info")) or []
+            except Exception:
+                _LOGGER.exception("Failed to load adviser metadata")
+
         return {
             "pipeline": self.pipeline.to_dict(),
             "products": [product.to_dict() for product in self.iter_products()],
-            "stack_info": self._stack_info,
+            "stack_info": list(chain(stack_info, self._stack_info or [])),
```

Comment:
what about `stack_info + (self._stack_info or [])`

Reply:
Adjusted in the most recent commit :+1: